### ISSUE TITLE
Collab Method - Fork and Pull or single shared repo? 

### ISSUE THREAD

**Issue by SPTechnology** [ETCGrantsDAO/etc-grants-dao-website#51]
Just want to confirm your favoured collab method; should I fork and you can pull into the upstream repo or just make pull requests directly to the main? @bobsummerwill

**Comment by SPTechnology** [comment]
Never mind - seeing that everyone else is forking will do the same. Can't delete issues so I'll close this for now.

**Comment by bobsummerwill** [comment]
Yeah - just make PRs please against your forked repo and we will merge.

I've just been making Andrew's commits for him, because he is new to GH.
